src/VisualStudio/VisualBasic/Impl/Help/HelpKeywords.vb

```diff
@@ -1,94 +1,94 @@
-        Friend Shared PreprocessorIf As String = "vb.#If"
-        Friend Shared Region As String = "vb.#Region"
-        Friend Shared [Redim] As String = "vb.Redim"
-        Friend Shared ClassConstraint As String = "vb.ClassConstraint"
-        Friend Shared QuerySelect As String = "vb.QuerySelect"
-        Friend Shared QuerySkip As String = "vb.QuerySkip"
-        Friend Shared QuerySkipWhile As String = "vb.QuerySkipWhile"
-        Friend Shared QueryTake As String = "vb.QueryTake"
-        Friend Shared QueryTakeWhile As String = "vb.QueryTakeWhile"
-        Friend Shared Negate As String = "vb.Negate"
-        Friend Shared StructureConstraint As String = "vb.StructureConstraint"
-        Friend Shared QueryWhere As String = "vb.QueryWhere"
-        Friend Shared XmlLiteralCdata As String = "vb.XmlLiteralCdata"
-        Friend Shared XmlLiteralComment As String = "vb.XmlLiteralComment"
-        Friend Shared XmlLiteralDocument As String = "vb.XmlLiteralDocument"
-        Friend Shared XmlLiteralElement As String = "vb.XmlLiteralElement"
-        Friend Shared XmlEmbeddedExpression As String = "vb.XmlEmbeddedExpression"
-        Friend Shared XmlLiteralProcessingInstruction As String = "vb.XmlLiteralProcessingInstruction"
-        Friend Shared NewConstraint As String = "vb.NewConstraint"
-        Friend Shared AnonymousKey As String = "vb.AnonymousKey"
-        Friend Shared AnonymousType As String = "vb.AnonymousType"
-        Friend Shared ObjectInitializer As String = "vb.ObjectInitializer"
-        Friend Shared ExtensionMethods As String = "vb.ExtensionMethods"
-        Friend Shared Nullable As String = "vb.Nullable"
-        Friend Shared AutoProperty As String = "vb.AutoImplementedProperty"
-        Friend Shared XmlDocComment As String = "vb.XmlDocComment"
-        Friend Shared [Rem] As String = "vb.Rem"
-        Friend Shared QueryFromIn As String = "vb.QueryFromIn"
-        Friend Shared QueryFromLet As String = "vb.QueryFromLet"
-        Friend Shared XmlPropertyChildAxis As String = "vb.XmlPropertyChildAxis"
-        Friend Shared XmlPropertyDescendantsAxis As String = "vb.XmlPropertyDescendantsAxis"
-        Friend Shared XmlPropertyAttributeAxis As String = "vb.XmlPropertyAttributeAxis"
-        Friend Shared XmlPropertyExtensionValue As String = "vb.XmlPropertyExtensionValue"
-        Friend Shared MyNamespaceKeyword As String = "vb.My"
-        Friend Shared CollectionInitializer As String = "vb.CollectionInitializer"
-        Friend Shared VarianceIn As String = "vb.VarianceIn"
-        Friend Shared VarianceOut As String = "vb.VarianceOut"
-        Friend Shared TaskListUserComments As String = "vbc0"
-        Friend Shared Main As String = "vb.Main"
-        Friend Shared Async As String = "vb.Async"
-        Friend Shared Iterator As String = "vb.Iterator"
-        Friend Shared Await As String = "vb.Await"
-        Friend Shared Yield As String = "vb.Yield"
+        Friend Const Assign As String = "vb.Assign"
```

Comment:
can we use shared-readonly instead?

Reply:
➡️ As I mentioned above, this is currently the subject of an open design discussion which we'll make a decision on and report back. To avoid unnecessary back and forth it didn't make sense to request a change until the design review reaches a conclusion.